assets/src/App.tsx

```diff
@@ -3,13 +3,16 @@ import { RouterProvider } from "react-router-dom";
 import { DeveloperInfoProvider } from "./contexts/DeveloperInfoContext";
 import { router } from "./Routes";
 import { UserProvider } from "./contexts/UserContext";
+import { ToastProvder } from "./features/shared/context/ToastContext";
 
 const App: FC = () => {
   return (
     <React.StrictMode>
       <UserProvider>
         <DeveloperInfoProvider>
-          <RouterProvider router={router} />
+          <ToastProvder>
+            <RouterProvider router={router} />
+          </ToastProvder>
```

Comment:
There are already 3 context providers. Maybe it will be good to make this more generic?

A ready-made solution is provided on [junior developer's first go-to site](https://stackoverflow.com/questions/51504506/too-many-react-context-providers)